commit 3fff5c88b273072e3657d37faed1618ac9bf48e8 (refs/changes/07/1607/1)
Author: YangYong3 <yy1111.yang@samsung.com>
Date:   2017-03-03T17:33:06+09:00 (2 years, 7 months ago)
    
    joyent/node-triton#128 triton ls state=provisioning` doesn't filter on state

diff --git a/lib/common/validation.js b/lib/common/validation.js
index 374db7c..4d24b3c 100644
--- a/lib/common/validation.js
+++ b/lib/common/validation.js
@@ -62,7 +62,8 @@ var VALID_VM_STATES = [
     'running',
     'stopped',
     'active',
-    'destroyed'
+    'destroyed',
+    'provisioning'
 ];
 
 var DEFAULT_QUOTA = 10; // GiB
